runtime/dolphin/src/lib.rs

```diff
@@ -708,6 +713,9 @@ construct_runtime!(
         Assets: pallet_assets::{Pallet, Call, Storage, Event<T>} = 45,
         AssetManager: pallet_asset_manager::{Pallet, Call, Storage, Config<T>, Event<T>} = 46,
         MantaPay: pallet_manta_pay::{Pallet, Call, Storage, Event<T>} = 47,
+
+        // Benchmark
+        StorageBench: pallet_storage_benchmark::{Pallet, Call, Storage, Event<T>} = 48,
```

Comment:
We should make testnet pallets have a high index by convention, maybe something like `201`.
That way we 
- can write a CI that checks calamari/manta don't have any of these
- don't run into issues with dol having a number we'd want to use for a grouped pallet (e.g. in the consensus or democracy group of ten) occupied by a private pallet